RecoLocalCalo/HcalRecProducers/python/hfQIE10Reco_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+hfQIE10Reco = cms.EDProducer("HcalSimpleReconstructor",
+    correctionPhaseNS = cms.double(0.0),
+    digiLabel = cms.InputTag("simHcalUnsuppressedDigis","HFQIE10DigiCollection"),
```

Comment:
On 2/11/16 1:10 PM, Kevin Pedro wrote:
> In RecoLocalCalo/HcalRecProducers/python/hfQIE10Reco_cfi.py
> <https://github.com/cms-sw/cmssw/pull/13232#discussion_r52667143>:
> 
>> @@ -0,0 +1,15 @@
>> +import FWCore.ParameterSet.Config as cms
>> +
>> +hfQIE10Reco = cms.EDProducer("HcalSimpleReconstructor",
>> +    correctionPhaseNS = cms.double(0.0),
>> +    digiLabel = cms.InputTag("simHcalUnsuppressedDigis","HFQIE10DigiCollection"),
> 
> This cfi and the corresponding algorithm in HcalSimpleReconstructor are
> just placeholders to demonstrate the use of the QIE10DigiCollection.
> This isn't in the RECO workflow for any scenario and never will be.
> Eventually it will be supplanted by the real algorithms, once they are
> developed.

I had another question in the comment: do we have an operational
digi2raw (packer) for simulation?

> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/13232/files#r52667143>.
>